File: backend/Enzivor.Api/Models/Dtos/ShowLandfillDto.cs
namespace Enzivor.Api.Models.Dtos
{
    
        public class ShowLandfillDto
        {
            // Basic info
            public int Id { get; set; }                  // Unique identifier
            public string? Name { get; set; }            // Known landfill name (if available)

            // Location & region
            public string RegionKey { get; set; } = string.Empty; // e.g. "Vojvodina", "Belgrade"
            public double Latitude { get; set; }                  // Center latitude
            public double Longitude { get; set; }                 // Center longitude

            // Classification & size
            public string Type { get; set; } = string.Empty;      // sanitary | unsanitary | wild
            public string Size { get; set; } = string.Empty;      // small | medium | large
            public int YearCreated { get; set; }

            // Optional calculated info
            public double? AreaM2 { get; set; }                   // Surface area if available
        }
 
}

File: envizor_client/EnvizorFrontend/src/app/DTOs/showLandfill-DTO.ts
export interface ShowLandfillDto {
  id: number; // Unique identifier
  name?: string; // Known landfill name (if available)

  regionKey: string; // e.g. "Vojvodina", "Belgrade"
  latitude: number; // Center latitude
  longitude: number; // Center longitude

  type: string; // sanitary | unsanitary | wild
  size: string; // small | medium | large
  yearCreated: number;

  areaM2?: number; // Surface area if available
}
